test/unit/ready.js

```diff
@@ -1,73 +1,70 @@
-module("event");
+module( "event" );
 
-(function(){
+(function() {
```

Comment:
Generally, it was asked to not fix code-style issues if PR is not about a code-style